cli/command/container/list.go

```diff
@@ -77,8 +77,7 @@ func buildContainerListOptions(opts *psOptions) (*types.ContainerListOptions, er
 		options.Limit = 1
 	}
 
-	options.Size = opts.size
-	if !options.Size && len(opts.format) > 0 {
+	if !opts.quiet && !options.Size && len(opts.format) > 0 {
```

Comment:
Moved the `quiet` check here, so that this parsing is never called if `--quiet` is set (in which case it's not needed)